web/tests/functional/report_viewer_api/test_hash_clash.py

```diff
@@ -140,15 +142,19 @@ def test_hash_clash(self):
                 fileid2 = f.fileId
 
         by_file = defaultdict(int)
+        by_checker_message = defaultdict(int)
+        by_bug_report_hash = defaultdict(int)
+
         for report in reports:
             by_file[report.fileId] += 1
+            by_checker_message[report.checkerMsg] += 1
+            by_bug_report_hash[report.bugHash] += 1
 
-        self.assertEqual(by_file[fileid1], 4)
+        self.assertEqual(by_file[fileid1], 5)
         self.assertEqual(by_file[fileid2], 1)
 
-        by_checker_message = defaultdict(int)
-        for report in reports:
-            by_checker_message[report.checkerMsg] += 1
-
-        self.assertEqual(by_checker_message['checker message'], 4)
+        self.assertEqual(by_checker_message['checker message'], 5)
         self.assertEqual(by_checker_message['checker message 2'], 1)
+
+        self.assertEqual(by_bug_report_hash['11111'], 5)
```

Comment:
The other 6 report entries in the list file already, having the `11111` as `issue_hash_content_of_line_in_context`.
We assert 5 instead of 6, because the second report entry is an exact copy of the first one, thus we need to ignore them (since they should result in the exact same path hash).